.changeset/gold-flies-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'fingerprint-pro-server-api-openapi': patch
+---
+
+**events-search**: Fixed vpn_confidence enum value
```

Comment:
Could you improve this message to say that we mean query param?

Reply:
@ilfa @TheUnderScorer

Do you think this is too verbose or ok?

```
*events-search**: Fixed `vpn_confidence` query parameter enum value formatting (`high,` -> `high`)
```